src/components/home-card/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { Card, Button } from '../base';
+import { newChallenge } from '../../services/challenge';
+const HomeCard = () => {
+  const history = useHistory();
+  const handleClick = () => {
+    newChallenge().then(resp => {
```

Comment:
Just suggestion 
```
const handleClick = () => newChallenge()
    .then(resp => history.push(`/challenge/${resp.auth.edition}`));
```

Reply:
the current `eslint/prettier` configuration force me to do it in that way but we can change this config later 🤔